src/assets/data/boardCharter.json

```diff
@@ -13,7 +13,7 @@
 	 },
 	 {"subSectionId": "member_backgrounds",
           "subSectionTitle": "Board Member Backgrounds",
-	  "subSectionParagraphs": ["Board members are not limited to but traditionally fit into one or more of the following categories:<ul><li><b>Technical Implementers</b> provide input and guidance regarding the creation, design, review, maintenance, and applications of CVE List records (<a href='/ResourcesSupport/FAQs#pc_cve_recordswhat_is_cve_record' target='_blank'>CVE Records</a>). This may include individuals who integrate CVE Records into products, such as content and development engineers working for product vendors, and others who consume CVE Records.</li><li><b>Subject Matter Experts (SMEs)</b> represent a significant constituency related to &mdash; or affected by &mdash; CVE, and are domain experts in the vulnerability management and reporting field. These members may include representatives from product vendors who represent the needs of their company, such as PSIRT team members, or product managers and product strategists who are representing customers.</li><li><b>Advocates</b> actively support or promote CVE in a highly visible fashion. These individuals are respected leaders within the security community who help bring credibility to CVE and give it a wider reach outside of the security community.</li></ul>"],
```

Comment:
Each word family change is its own commit. I'm happy to drop any commits that people find objectionable.